src/components/FilmList/Films.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import './Films.scss';
+import { isEmptyObject, isNonEmptyArray } from './deps';
+import { ListElement } from './type';
+
+class Films extends React.Component<ListElement> {
+    state = {
+        isShow: false,
+    };
+    onClick = (): void => {
+        this.setState({
+            isShow: !this.state.isShow,
+        });
+    }
+    render() {
+        const addClass = this.state.isShow ? "show" : "hide";
+        const renderRatings = (props: any) => {
+            return props.map((props: any, key: any) => {

```

Comment:
Shadowed variable "props" makes the code hard to read.
https://palantir.github.io/tslint/rules/no-shadowed-variable/